tslint.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
is tslint.kson supposed to be commited to the repo?

Reply:
Yes. This contains rules about the codestyle of the Typescript source files, it will be useful because then you can have a ci task run the lint check and verify that everything is ok.